app/views/internal/classified_listings/index.html.erb

```diff
@@ -11,7 +11,12 @@
         </div>
         <div class="form-group">
           <%= label_tag(:filter, "Category") %>
-          <%= select_tag(:filter, options_for_select([""] + ClassifiedListing.categories_available.keys, params[:filter])) %>
+          <%= select_tag(:filter, options_for_select(ClassifiedListing.categories_available.keys, params[:filter]), include_blank: true) %>
+        </div>
+        <div class="form-group">
+          <%= label_tag(:published, "Published only") %>
+          <%= hidden_field_tag(:published, 0) %>
+          <%= check_box_tag(:published, 1, true) %>
```

Comment:
If I uncheck the checkbox and click "filter", the checkbox will be checked after the filtering. I would expect it to remain unchecked after unchecking. How do you feel about making it checked or unchecked based on the corresponding param (when it exists)?

Reply:
The reason I did that was as a workflow optimization for @michael-tharrington. My rationale:

1. Most of the time he wants to see the filtered list.
2. If he really wants to see all listings he can uncheck the box and hit "filter".
3. If he wants to get back to the filtered list, he only needs to press "filter" instead of fiddling with the check box. 

I see how it's confusing though, so will change it.